src/js/angular/rest/ttyg.rest.service.fake.backend.js

```diff
@@ -98,22 +102,27 @@ export class TtygRestServiceFakeBackend {
     }
 
     getAgents() {
-        return Promise.resolve({data: [...agentsList]});
+        return new Promise((resolve) => setTimeout(() => resolve({data: []}), LOAD_AGENTS_DELAY));
```

Comment:
We should return agentsList, not an empty list.

Reply:
i've forgotten to revert this here but I did it in my next MR